libraries/AP_AHRS/AP_AHRS.cpp

```diff
@@ -3652,6 +3652,50 @@ float AP_AHRS::get_air_density_ratio(void) const
     return 1.0 / sq(eas2tas);
 }
 
+void AP_AHRS::set_data_sending_state(DATA_SENDING_STATE data_sending_state)
```

Comment:
```suggestion
void AP_AHRS::set_data_sending_state(DataSendingState new_state)
```

similarly for other enum names

Reply:
Done